include/openPMD/Mesh.hpp

```diff
@@ -182,6 +182,14 @@ class Mesh : public BaseRecord< MeshRecordComponent >
 
     void flush_impl(std::string const&) override;
     void read() override;
+
+    /**
+     * @brief Verify that a mesh in a closed iteration has not
+     *        been wrongly accessed.
+     *
+     * @return true If closed iteration had no wrong accesses.
+     * @return false Otherwise.
+     */
```

Comment:
```suggestion
     * @todo needs to be implemented
     */
```

Reply:
is it intentional that particles already have the method but meshes, don't?

Reply:
I implemented the method on the base class instead (i.e. `BaseRecord`) since it also occurs somewhere in the hierarchy for particles. This is dead code, I removed it now.